x11-terms/st-terminfo/st-terminfo-0.8.4.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=${PN%-*}
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="Terminfo for st"
+HOMEPAGE="https://st.suckless.org/"
+SRC_URI="https://dl.suckless.org/st/${MY_P}.tar.gz"
+
+LICENSE="MIT-with-advertising"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc64 ~x86"
+RESTRICT="test"
+
+DEPEND=">=sys-libs/ncurses-6.0:0="
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	mkdir terminfo
+	default
+}
+
+src_configure() {
+	return
```

Comment:
we usually use `:` as shell true in such cases instead of return.

Reply:
Replaced!